Source/WebCore/accessibility/AccessibilityListBox.cpp

```diff
@@ -115,10 +115,11 @@ AXCoreObject::AccessibilityChildrenVector AccessibilityListBox::visibleChildren(
     if (!childrenInitialized())
         addChildren();
     
+    const auto& children = const_cast<AccessibilityListBox*>(this)->unignoredChildren(/* updateChildrenIfNeeded */ false);
     AccessibilityChildrenVector result;
-    for (unsigned i = 0; i < m_children.size(); i++) {
+    for (unsigned i = 0; i < children.size(); i++) {
```

Comment:
can you. move the .size() call outside the loop so we only have to call one time?